src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
+    return <PersonLink person={person} people={people} key={person.slug} />;
+  });
```

Comment:
The `map` function is being used to transform the `people` array into an array of `PersonLink` components. However, this logic should be placed directly within the JSX in the `tbody` to keep the component clean and more readable. Instead of assigning it to a variable, you can directly map over `people` inside the JSX.